File: src/main/java/org/ma99us/sbda/model/GpioPin.java
package org.ma99us.sbda.model;

import lombok.AllArgsConstructor;
import lombok.Data;

@Data
@AllArgsConstructor
public class GpioPin {
    private int num;
    private Integer gpio;
    private Integer wPi;
    private String name;
    private String mode;
    private Integer v;
}

File: src/main/webapp/src/model/gpio-pin.ts
export class GpioPin {
  num?: number
  gpio?: number;
  wPi?: number;
  name?: string;
  mode?: number;
  v?: number;
}

export class PinAction {
  gpio?: number;
  wPi?:  number;
  isHigh?: boolean;
  mode?: string;
  durationMs?: number;

  readAll: boolean = false;
}